client/src/sessions/authorization_area/s0.rs

```diff
@@ -0,0 +1,34 @@
+use super::AuthorizationArea;
+use crate::sessions::PasswordSession;
+
+impl AuthorizationArea<PasswordSession, PasswordSession, PasswordSession> for () {
```

Comment:
I'm not the biggest fan of us re-using PasswordSession here, I wonder if we should have a `NoSession` special type or something. Just seems a bit weird to re-use `PasswordSession` for this purpose.

Reply:
you are right! I created a specific `NotSession` type for that, feel free to review that here https://github.com/tpm-rs/tpm-rs/pull/95/commits/89dba53622a82d6b79f1a8810b52471b0a03b5f4